### DISCUSSION TITLE
Does KEDA force sequential upgrades?

### DISCUSSION THREAD

**Discussion by qpetraroia** [kedacore/keda#5966] [Q&A / Need Help] [answered]
Does KEDA force users to do sequential upgrades on their KEDA versions? For example, if I am on KEDA version 2.11. Can I upgrade to 2.14? Or do I need to go version by version?

Thanks!

**Comment by tomkerkhove** [comment]
As far as I know, it does not but maybe @zroubalik / @JorTurFer tell me otherwise?

**Reply by JorTurFer** [reply, marked as answer]
You should be able to upgrade multiple versions at once if there isn't any breaking change that affects to your use case (e.g: prometheus metrics relabelling)
